.github/workflows/workflows.yaml

```
@@ -85,6 +85,22 @@ jobs:
         uses: actions/setup-go@main
         with:
           go-version: 1.23
+        run: |
```

actionlint (1.7.4)

this step is for running action since it contains at least one of "uses", "with" keys, but also contains "run" key which is used for running shell command

(syntax-check)